common/models/battle-log-model.ts

```diff
@@ -242,6 +242,7 @@ export class BattleLogModel {
 				damage: `$b${subAttack.calculateDamage() + weaknessDamage}hp$`,
 				defaultLog: this.generateEffectEntryHeader(singleUse),
 				coinFlip: this.generateCoinFlipMessage(attack, coinFlips),
+				attack: subAttack,
```

Comment:
Where are you reading this property?

Reply:
the default hermit attack log, and Ren and Cleo